src/packages/client/src/runtime/getPrismaClient.ts

```diff
@@ -784,16 +800,48 @@ export function getPrismaClient(config: GetPrismaClientOptions): any {
         delegate.count = (args) =>
           clients[mapping.model]({
             operation: 'query',
-            actionName: 'count',
+            actionName: 'count', // actionName is just cosmetics 💅🏽
             rootField: mapping.aggregate,
             args: args
               ? {
-                  select: { count: args },
+                  ...args,
+                  select: { count: true },
                 }
               : undefined,
             dataPath: ['count'],
           })
 
+        delegate.aggregate = (args) => {
+          /**
```

Comment:
I might be wrong but I guess we should be checking here for experimental flags here like we are doing for transaction and throwing an error if the flag is not there.

Like it is done here: https://github.com/prisma/prisma/blob/7fe8bd188d42860387c5e62030b7ba57322c2dcc/src/packages/client/src/runtime/getPrismaClient.ts#L592

Reply:
See above comment